packages/openid4vp/src/openid4vp-auth-response/verify-openid4vp-auth-response.ts

```diff
@@ -0,0 +1,97 @@
+import { Oauth2Error } from '@openid4vc/oauth2'
+import type { Openid4vpAuthRequest } from '../openid4vp-auth-request/v-openid4vp-auth-request'
+import {
+  parseDcqlPresentationFromVpToken,
+  parsePresentationsFromVpToken,
+} from '../vp-token/parse-presentations-from-vp-token'
+import type { Openid4vpAuthResponse } from './v-openid4vp-auth-response'
+import type { VerifyOpenid4VpAuthorizationResponseResult } from './verify-openid4vp-auth-response-result'
+
+/**
+ * The following steps need to be done manually
+    // validating the id token
```

Comment:
This is explicity not supported, so doesn't have to be performend manuallly right now (as we throw an error?)